client/src/components/teams/MembersToBeInvited.js

```diff
@@ -0,0 +1,67 @@
+import {
+  Typography,
+  Grid,
+  Button,
+  makeStyles,
+  InputLabel,
+} from "@material-ui/core";
+import { Brightness1 } from "@material-ui/icons";
+import React from "react";
+
+const MembersToBeInvited = ({ teamMembers, setTeamMembers }) => {
+  const classes = useStyles();
+  const teamMemberNotEmpty = teamMembers.length !== 0;
+  const removeEmailFromInvitedList = (memberToRemove) => () => {
+    // Caveat: each email must be unique in the list for this to work
+    const filteredList = teamMembers.filter(
+      (member) => member.email !== memberToRemove.email
+    );
```

Comment:
Caveat makes sense. The way to handle that would be to just not allow adding duplicate emails, which is doable because you did a good job to organize the structure so both components are passed the `teamMembers` as a prop.